cv_next/app/feed/components/CVItem.tsx

```diff
@@ -50,37 +50,41 @@ export default function CVItem({ cv }: CVCardProps) {
   );
 
   useEffect(() => {
+    let isMounted = true;
```

Comment:
Are you certain this locally-scoped var is giving you what you need? It seems suspicious to me. I think it'd be preferrable if you'd turn this into a ref variable.

Reply:
Also if all you want is for this effect to run only during mount, just empty the deps array

Reply:
I want it to run only once but I want to use the interval to revalidate the image every 2 minutes.
Reason I added this was that when I signed out or move to a different page, it would tell me that it can't find a server action with that ID

Reply:
I see, so the cleanup function should only run on unmount (or in legacy react terms, willUnmount)? Note that this is not the functionality you've implemented.
The cleanup function is called on every re-render. See https://react.dev/reference/react/useEffect#useeffect - "After every re-render with changed dependencies, React will first run the cleanup function (if you provided it) with the old values...".
What you want is to separate this into multiple effects, let's say first for the current version you have and a second for unmounting (only the return function with no deps).
BTW a better method instead of the `isMounted` check, imo, will be to take the interval ID as a ref and clear it on the unmounting effect (https://developer.mozilla.org/en-US/docs/Web/API/clearInterval).

Reply:
I think I understood what you meant, I uploaded a different version